### PR TITLE
feat: update ocm-controller to v0.13.0 and all controllers and the e2e framework

### PR DESCRIPTION
**Pull request by Skarlso**

### CODE REVIEW

**Comment by Skarlso** [comment]
Locally, it's running now. I have to update the components and things like that.

```
PASS
ok      github.com/open-component-model/mpas/e2e        347.975s

```

**Comment by Skarlso** [comment]
The CLI test will fail until I release a new version of the component it's using for bootstrapping.

**Review by phoban01 — APPROVED** [review]

File: internal/bootstrap/bootstrap.go [hunk 1]
```diff
@@ -477,6 +449,46 @@ func (b *Bootstrap) Run(ctx context.Context) error {
...
 	}
+
+	// Create Certificate secret if needed
+	if b.generateDevCertificate {
+		ca, key, err := GenerateDeveloperSelfSignedCertificate()
+		if err != nil {
+			return fmt.Errorf("failed to generate developer certificate: %w", err)
+		}
+
+		for _, namespace := range []string{env.DefaultOCMInstallPath, env.DefaultsNamespace} {
```
Comment (souleb) [inline internal/bootstrap/bootstrap.go:460]: I would like to move this in `internal/kubeutils` at some point. But looks good to me 👍

**Review by souleb — APPROVED** [review]
Awesome work. Thanks @Skarlso

**Pull request merged**